api/tests/Behat/features-v2/reporting/sections/assets/assets.feature

```diff
@@ -0,0 +1,51 @@
+@v2 @assets
+Feature: Assets
+
+@pfa-high-not-started
+  Scenario: A user has no assets to add
+    Given a Lay Deputy has not started a report
+    When I view and start the assets report section
+    And I confirm the client has no assets
+    Then I should see the expected assets report section responses
+    When I follow link back to report overview page
+    Then I should see "assets" as "no assets"
+
+@pfa-high-not-started
+  Scenario: A user adds a single asset
+    Given a Lay Deputy has not started a report
+    When I view and start the assets report section
+    And I confirm the client has assets
+    And I add a single asset
+    Then I should see the expected assets report section responses
+    When I follow link back to report overview page
+    Then I should see "assets" as "1 asset"
+
+@pfa-high-not-started
+  Scenario: A user adds a single property asset
+    Given a Lay Deputy has not started a report
+    When I view and start the assets report section
+    And I confirm the client has assets
+    And I add a single property asset
+    Then I should see the expected assets report section responses
+    When I follow link back to report overview page
+    Then I should see "assets" as "1 asset"
+
+@pfa-high-not-started
+  Scenario: A user adds multiple properties
+    Given a Lay Deputy has not started a report
+    When I view and start the assets report section
+    And I confirm the client has assets
+    And I add multiple property assets
+    Then I should see the expected assets report section responses
+    When I follow link back to report overview page
+    Then I should see "assets" as "3 assets"
+
+@pfa-high-not-started
+  Scenario: A user adds multiple assets and a property
+    Given a Lay Deputy has not started a report
+    When I view and start the assets report section
+    And I confirm the client has assets
+    And I add multiple assets and a property
```

Comment:
Same as a above - could be clearer to explicitly say how many we're adding so the last line matches up